### ISSUE TITLE
`cmake` is now at version 3.27.6

### ISSUE THREAD

**Issue by pfr-dev** [jstkdng/ueberzugpp#156]
I was able to build on Void linux after downgrading `cmake` to v3.18 as required.

Can this be updated so that more current versions of `cmake` can be used? this will make packaging for other distributions much easier. I wonder how the AUR package for Arch is built as i imagine that Arch uses the latest version of `cmake`. 

If it it possible to build with current versions of `cmake` can the build instructions please be updated?

cheers

**Comment by jstkdng** [comment]
`cmake` 3.18 is the minimum version required so it should build with any version greater than 3.18. Why are you unable to build with the latest cmake, does an error appear?

>I wonder how the AUR package for Arch is built as i imagine that Arch uses the latest version of cmake.

it works fine on arch with latest `cmake`, something could be wrong on your setup.

**Comment by 0323pin** [comment]
@jstkdng I think the reason for confusion is the README.MD here, it clearly states `cmake (3.18 <= )` under required dependencies. Was this a typo?

When digging in `CMakeLists.txt`, it becomes obvious that it should be "cmake (3.18 **>**= )"

**Comment by pfr-dev** [comment]
Ok, my bad. I initially got a `cmake` error when I first attempted to build, and after reading the README and seeing `cmake (3.18 <= )` I downgraded and tried again. The weird thing is I then got a different error which made me think that the downgrade was necessary. 

Nevertheless, it builds on `cmake 3.27.6_1` so I think perhaps the README needs to be updated because it looks as though building required a version 3.18 or lower. As Pin says above, it should read 3.18>=

**Comment by jstkdng** [comment]
>it becomes obvious that it should be "cmake (3.18 >= )"

uhh, 3.18 <= x , being x the cmake version means x is greater than or equal to 3.18.    3.18 >= x would mean the opposite (maximum version is 3.18). Is that how math works? or is in versioning different.

anyways, that can be fixed by using words, the cmake error is still unclear, please do provide it.

**Comment by 0323pin** [comment]
> uhh, 3.18 <= x , being x the cmake version means x is greater than or equal to 3.18. 3.18 >= x would mean the opposite (maximum version is 3.18). Is that how math works?

That's correct but, there's no **x** in the README.MD, so it becomes unclear. Maybe `cmake >= 3.18` would be easier to understand.

Anyway, I think @pfr-dev has solved this _non-issue_, apparently his initial error was unrelated to the `cmake` version.

Just to clarify, the only reason I'm here is because he has asked for a package.
